### PR TITLE
FRONT-47

### PR DESCRIPTION
**Pull request by Ekaterina-Bondareva**
[Hook up Login/Signup pages to adapters#47](https://github.com/Code-the-Dream-School/bb-practicum-team1-front/issues/47)

### CODE REVIEW

**Review by koral14 — APPROVED** [review]
Looks good to me! I pulled the branch, but got an 401. I don't know how to test it yet, therefore will leave one of our mentors to do it.  Great job Ekaterina!

File: src/adapters/auth-adapters.ts [hunk 1]
```diff
@@ -83,10 +83,12 @@ This function login a user with the provided information.
 */
 export const loginAdapter = async (loginInput: LoginInputs ) => {
     const url =`http://localhost:8000/api/v1/user/authentication`
-    const data = await fetchAPIData(url , 'POST', loginInput)
-    setCookie('shelf-share-session', JSON.stringify(data), 1)
+    const data = await fetchAPIData(url , 'POST', loginInput, false)
+    if (data) {
+        setCookie(cookieName, JSON.stringify(data.token), 1)
```
Comment (jpremo) [inline src/adapters/auth-adapters.ts:88]: I like the error handling you added in here. We do want all of data to be in the cookie though, so can you change that part back?

File: src/adapters/auth-adapters.ts [hunk 2]
```diff
@@ -83,10 +83,12 @@ This function login a user with the provided information.
...
+        console.log("Cookie set")
```
Comment (jpremo) [inline src/adapters/auth-adapters.ts:89]: Remove this log

File: src/adapters/auth-adapters.ts (hunk 1 above)
Comment (jpremo) [reply src/adapters/auth-adapters.ts:88]: Can you add a check for data for the signUpAdapter as well?

**Review by jpremo — CHANGES_REQUESTED** [review]
Seems to work well. Nice job! A few minor change requests.

File: src/adapters/auth-adapters.ts (hunk 2 above)
Comment (Ekaterina-Bondareva) [reply src/adapters/auth-adapters.ts]: done

File: src/adapters/auth-adapters.ts (hunk 1 above)
Comment (Ekaterina-Bondareva) [reply src/adapters/auth-adapters.ts]: done

**Review by jpremo — APPROVED** [review]

**Review by cecerodrgz — APPROVED** [review]

**Pull request merged**